examples/programs/anchor-feed-parser/src/lib.rs

```diff
@@ -1,17 +1,20 @@
 use anchor_lang::prelude::*;
 use anchor_lang::solana_program::clock;
+// use anchor_lang::{Discriminator, Owner, ZeroCopy};
+// use bytemuck::{Pod, Zeroable};
 use std::convert::TryInto;
-pub use switchboard_v2::{
-    AggregatorAccountData, SwitchboardDecimal, SWITCHBOARD_V2_DEVNET, SWITCHBOARD_V2_MAINNET,
-};
+pub use switchboard_v2::{AggregatorAccountData, SwitchboardDecimal, SWITCHBOARD_PROGRAM_ID};
 
 declare_id!("FnsPs665aBSwJRu2A8wGv6ZT76ipR41kHm4hoA3B1QGh");
 
 #[derive(Accounts)]
 #[instruction(params: ReadResultParams)]
 pub struct ReadResult<'info> {
-    /// CHECK:
-    pub aggregator: AccountInfo<'info>,
+    #[account(
+        constraint = 
+            *aggregator.to_account_info().owner == SWITCHBOARD_PROGRAM_ID @ FeedErrorCode::InvalidSwitchboardAccount
```

Comment:
AccountLoader already verifies the program owner of the account. The imported `AggregatorAccountData` struct knows what program is supposed to own it based on the `declare_id` program id.